environs/config/config.go

```diff
@@ -775,6 +769,10 @@ func Validate(_ctx context.Context, cfg, old *Config) error {
 
 	if v, ok := cfg.defined[ContainerNetworkingMethod].(string); ok {
 		switch v {
+		case "fan":
+			if cfg, err := cfg.FanConfig(); err != nil || cfg == nil {
+				return errors.New("container-networking-method cannot be set to 'fan' without fan-config set")
+			}
```

Comment:
Is this a mistake?

Reply:
Ops, this might have been removed recently in the main.